src/RadixDlt.NetworkGateway.Abstractions/Model/EntityRelationship.cs

```diff
@@ -62,29 +62,29 @@
  * permissions under this License.
  */
 
-using Microsoft.EntityFrameworkCore.Migrations;
+namespace RadixDlt.NetworkGateway.Abstractions.Model;
 
-#nullable disable
-
-namespace RadixDlt.NetworkGateway.PostgresIntegration.Migrations
+// TODO change to enum
```

Comment:
hmm I'm not sure, maintaining string might be easier. Also I'd make it more descriptive by nesting classes i.e
```
public static class EntityRelationship
{
    public static class Component
    {
        public const string Package
    }

    public static class Validator 
    {
    }

... etc
}
```

Reply:
Had we decided to go with strings that would be no doubt the way to go. However I have rather strong objections here. We should go with enum as
- those relationships are well-defined and finite; we can always add new ones if needed,
- enums are lighter and easier to work with